docs/user-guide/interfaces/cli/basics.md

```diff
@@ -0,0 +1,88 @@
+Before we get to using the CLI, let's cover some information on
+configuration and behavior.
+
+## Configuring
+
+The default Rug CLI configuration file is located at
+`~/.atomist/cli.yml`.  You can override the default location using the
+`--settings=PATH` command-line option.  As the configuration file
+extension suggests, it is a [YAML][yaml] file.
+
+[yaml]: http://yaml.org/
+
+If you are using the default configuration file and one does not
+exist, the CLI will create a default one for you with contents like
+the following.
+
+```yaml
+# Set up the path to the local repository
+local-repository:
+  path: "${user.home}/.atomist/repository"
+
+# Set up remote repositories to query for Rug archives. Additionally one of the
+# repositories can also be enabled for publication (publish: true).
+remote-repositories:
+  maven-central:
+    publish: false
+    url: "http://repo.maven.apache.org/maven2/"
+  rug-types:
+    publish: false
+    url: "https://atomist.jfrog.io/atomist/libs-release"
+  rugs:
+    publish: false
+    url: "https://atomist.jfrog.io/atomist/rugs-release"
+
+# Set up Rug catalog endpoints for searching
+catalogs:
+- "https://api.atomist.com/catalog"
+```
```

Comment:
The catalog urls are not a root level element any longer. In fact per default we don't this at all. You can remove that part. Here is current default `cli.yml`:

```# Set up the path to the local repository
local-repository:
  path: "${user.home}/.atomist/repository"

# Set up remote repositories to query for Rug archives. Additionally one of the
# repositories can also be enabled for publication (publish: true).
remote-repositories:
  central:
    publish: false
    url: "http://repo.maven.apache.org/maven2/"
  rugs:
    publish: false
    url: "https://atomist.jfrog.io/atomist/rugs"
```